bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAlarmHandler.java

```diff
@@ -270,14 +264,13 @@ private void updateZoneAlarm(WhatAlarm w) {
 
     private void updateZoneAlarmTimeStamp() {
         ZonedDateTime now = ZonedDateTime.now();
-        ChannelUID cuid = new ChannelUID(getThing().getUID(), CHANNEL_ALARM_ZONE_TIMESTAMP);
-        updateState(cuid, new DateTimeType(now.truncatedTo(ChronoUnit.SECONDS)));
+        updateState(CHANNEL_ALARM_ZONE_ALARM_TIMESTAMP, new DateTimeType(now.truncatedTo(ChronoUnit.SECONDS)));
```

Comment:
```suggestion
        updateState(CHANNEL_ALARM_ZONE_ALARM_TIMESTAMP, new DateTimeType());
```

Reply:
line 266 is unuseful at this point so  I would remove it and  as a consequence   would  updateZoneTimeStamp() implementation still convenient? I could  simply invoke  updateState(CHANNEL_ALARM_ZONE_ALARM_TIMESTAMP, new DateTimeType())  directly at line 228...

Reply:
done as per previous comment